resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        {
+          "code": "LIFE_EVENT_ALREADY_EXISTS",
+          "message": "The investor’s life event has already been reported",
+          "lifeEventId": "3456789000"
+        }
+        </pre>
+                                                                                                </td>
+                                                                                                </tr> 
+            </thead>
+            </tbody>
+        </table>
+      operationId: requestReleaseOfFundsToBuyProperty
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/authorizationHeader'
+      - $ref: '#/components/parameters/lisaManagerReferenceNumber'
+      - $ref: '#/components/parameters/accountId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/lifetimeIsaManagerAccountsEventsFundReleasesRequest'
+            examples:
+              example-1:
+                value:
+                  eventDate: 2017-05-10
+                  withdrawalAmount: 4000
+                  conveyancerReference: CR12345-6789
+                  propertyDetails:
+                    nameOrNumber: '1'
+                    postalCode: AA11 1AA
+              example-2:
+                value:
+                  eventDate: 2017-05-05
+                  withdrawalAmount: 5000
+                  supersede:
+                    originalLifeEventId: '3456789000'
+                    originalEventDate: 2017-05-10
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaManagerAccountsEventsFundReleasesResponse'
+              examples:
+                lifeEventCreated:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Fund release created
+                      lifeEventId: '3456789000'
+                lifeEventSuperseded:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Fund release superseded
+                      lifeEventId: '3456789001'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                invalidLmrn:
+                  $ref: '#/components/examples/invalidLmrn'
+                invalidAccountId:
+                  $ref: '#/components/examples/invalidAccountId'
+                invalidPayLoad:
```

Comment:
Please add missing description

```
invalidPayLoad:
  description: Enter a payment which is not valid
  value:
    code: INVALID_PAYLOAD
    message: Submission has not passed validation
```

Reply:
This description in raml looks a bit funny.  I would rather follow convention and use the same description as the message? Is that ok?

Reply:
Added `description: Submission has not passed validation`